### PR TITLE
APIGOV-27137 - metric logger and updates

### PR DESCRIPTION
**Pull request by jcollins-axway**

### CODE REVIEW

**Review by sbolosan — APPROVED** [review]

**Review by sbolosan — DISMISSED** [review]

File: pkg/config/logconfig.go [hunk 1]
```diff
@@ -70,12 +89,24 @@ func AddLogConfigProperties(props properties.Properties, defaultFileName string)
 	props.AddStringProperty(pathLogFileName, defaultFileName, "Name of the log files")
 	props.AddStringProperty(pathLogFilePath, "logs", "Log file path if output type is file or both")
 	props.AddIntProperty(pathLogFileMaxSize, 10485760, "The maximum size of a log file, in bytes  (default: 10485760 - 10 MB)")
-	props.AddIntProperty(pathLogFileMaxAge, 0, "The maximum number of days, 24 hour periods, to keep the log file backps")
+	props.AddIntProperty(pathLogFileMaxAge, 0, "The maximum number of days, 24 hour periods, to keep the log file backips")
```
Comment (dgghinea) [inline pkg/config/logconfig.go:92]: backups*

File: pkg/traceability/healthcheck_test.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
+package traceability
```
Comment (dgghinea) [inline pkg/traceability/healthcheck_test.go:1]: I believe this should not be commented out

**Review by dgghinea — DISMISSED** [review]

**Review by vivekschauhan — DISMISSED** [review]

File: pkg/traceability/healthcheck_test.go (hunk 2 above)
Comment (jcollins-axway) [reply pkg/traceability/healthcheck_test.go:1]: you right, will fix

**Review by sbolosan — DISMISSED** [review]

**Review by sbolosan — DISMISSED** [review]

**Review by sbolosan — DISMISSED** [review]

**Review by sbolosan — DISMISSED** [review]

**Comment by jcollins-axway** [comment]
@vivekschauhan @sbolosan I think this is truly ready for review.

**Pull request merged**